aiida_common_workflows/workflows/relax/quantum_espresso/extractors.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+Post-processing extractor functions for the `QuantumEspressoCommonRelaxWorkChain`.
+"""
+
+from .workchain import QuantumEspressoCommonRelaxWorkChain
+
+
+def get_ts_energy(common_relax_workchain: QuantumEspressoCommonRelaxWorkChain) -> float:
+    """
+    Return the TS value of a concluded QuantumEspressoCommonRelaxWorkChain.
+    T the fictitious temperature due to the presence of a smearing and S is
+    the entropy.
+
+    :param common_relax_workchain: Quantum ESPRESSO common relax work chain for which to extract the smearing energy.
+
+    :returns: Smearing energy in eV.
+    """
+    from aiida.common import LinkType
+    from aiida.orm import WorkChainNode
+    from aiida.plugins import WorkflowFactory
+
+    if not isinstance(common_relax_workchain, WorkChainNode):
+        return ValueError('The input is not a workchain (instance of `WorkChainNode`)')
+    if common_relax_workchain.process_class != WorkflowFactory('common_workflows.relax.quantum_espresso'):
+        return ValueError('The input workchain is not a `QuantumEspressoCommonRelaxWorkChain`')
```

Comment:
```suggestion
from aiida.plugins import WorkflowFactory

QuantumEspressoCommonRelaxWorkChain = WorkflowFactory('common_workflows.relax.quantum_espresso')


def get_ts_energy(common_relax_workchain: QuantumEspressoCommonRelaxWorkChain) -> float:
    """Return the TS value of a concluded QuantumEspressoCommonRelaxWorkChain.

    T the fictitious temperature due to the presence of a smearing and S is the entropy.

    :param common_relax_workchain: Quantum ESPRESSO common relax work chain for which to extract the smearing energy.

    :returns: The TS value in eV.
    """
    from aiida.common import LinkType
    from aiida.orm import WorkChainNode

    if not isinstance(common_relax_workchain, WorkChainNode):
        return ValueError('The input is not a workchain (instance of `WorkChainNode`)')

    if common_relax_workchain.process_class != QuantumEspressoCommonRelaxWorkChain:
        return ValueError('The input workchain is not a `QuantumEspressoCommonRelaxWorkChain`')
```

Reply:
Oh wait, forgot to integrate these comments, just a second @sphuber

Reply:
Done! Note that I don't use the `WorkflowFactory` here. I'm not a huge fan of the factories, since they require me to type/copy the class instead of just using tab-completion, fail to allow tab-completion for methods unless I add type hinting, and don't allow me to use tools to quickly find the definition of the class.